### DISCUSSION TITLE
Please be more clear on this vscode extension

### DISCUSSION THREAD

**Discussion by jitvimol** [34j/vscode-black#21] [Q&A] [answered]
I believe that there are some people mistaken using this extension instead of original black formatter one. 

If you intend to use the name and the code has partial integrate with the original, you should write more clear instruction on detail usage.

Raise clear purpose of what this extension does and how to use it.  Not simple make people misunderstood and install the extension.  please kindly take it into consideration.  Thanks

**Comment by 34j** [comment, marked as answer]
Thank you for your suggestion. However

> original black formatter one

Unfortunately, Black is a Python package and has nothing to do with VSCode.

> the code has partial integrate with the original

The code is independent actually

This package was made for myself in the first place. There is no intent to mislead you. I would also refrain from writing about it as it has nothing to do with other extensions. Thank you in advance for your understanding.
